diff --git a/content/public/common/url_constants.h b/content/public/common/url_constants.h
index 6c03bc3be9d3c..3f3a7e661f9d8 100644
--- a/content/public/common/url_constants.h
+++ b/content/public/common/url_constants.h
@@ -19,6 +19,7 @@ namespace content {
 inline constexpr char kChromeDevToolsScheme[] = "devtools";
 inline constexpr char kChromeErrorScheme[] = "chrome-error";
 inline constexpr char kChromeUIScheme[] = "chrome";  // Used for WebUIs.
+inline constexpr char kAstroUIScheme[] = "astro";
 inline constexpr char kChromeNativeScheme[] = "chrome-native";
 inline constexpr char kChromeUIUntrustedScheme[] = "chrome-untrusted";
 inline constexpr char kViewSourceScheme[] = "view-source";
