diff --git a/chrome/browser/prefs/browser_prefs.cc b/chrome/browser/prefs/browser_prefs.cc
--- a/chrome/browser/prefs/browser_prefs.cc
+++ b/chrome/browser/prefs/browser_prefs.cc
@@ -1723,6 +1723,10 @@
   registry->RegisterStringPref("oxy.token_expiry", "");

+  // Oxy ad blocker prefs
+  registry->RegisterBooleanPref("oxy.adblock.enabled", true);
+  registry->RegisterDictionaryPref("oxy.adblock.site_overrides");
+
   dom_distiller::DistilledPagePrefs::RegisterProfilePrefs(registry);
   DownloadPrefs::RegisterProfilePrefs(registry);
